response/response.go

```diff
@@ -67,7 +67,7 @@ func New(code int, message string, data *Data) *Response {
 
 // SQLError returns a prepared 204 No Content response if the error passed is of type sql.ErrNoRows,
 // otherwise, returns a 500 Internal Server Error prepared response.
-func SQLError(err error) *Response {
+func SQLError(format string, err error) *Response {
```

Comment:
I reckon we can get best of both worlds here and have the two functions:
`SQLErrorf(format string, err error)`
`SQLError(err error)`

Where SQLError (non formattable) just wraps Errorf with the default format

Reply:
Yeah, I like that...  I'll make a change.